wzdx/raw_to_standard/navjoy_568.py

```diff
@@ -7,102 +7,119 @@
+        "mile_marker_start": "mileMarkerStart2",
+        "mile_marker_end": "mileMarkerEnd2",
+        "directions_of_traffic": "directionOfTraffic2",
+        "current_posted_speed": "currentPostedSpeed2",
+        "reduced_speed_limit": "requestedTemporarySpeed2",
+        "start_date": "workStartDate2",
+        "end_date": "workEndDate2",
     },
     {
-        'street_name': 'streetNameFrom3',
-        'mile_marker_start': 'mileMarkerStart3',
-        'mile_marker_end': 'mileMarkerEnd3',
-        'directions_of_traffic': 'directionOfTraffic3',
-        'current_posted_speed': 'currentPostedSpeed3',
-        'reduced_speed_limit': 'requestedTemporarySpeed3',
-        'start_date': 'workStartDate3',
-        'end_date': 'workEndDate3',
+        "street_name": "streetNameFrom3",
+        "mile_marker_start": "mileMarkerStart3",
+        "mile_marker_end": "mileMarkerEnd3",
+        "directions_of_traffic": "directionOfTraffic3",
+        "current_posted_speed": "currentPostedSpeed3",
+        "reduced_speed_limit": "requestedTemporarySpeed3",
+        "start_date": "workStartDate3",
+        "end_date": "workEndDate3",
     },
     {
-        'street_name': 'streetNameFrom4',
-        'mile_marker_start': 'mileMarkerStart4',
-        'mile_marker_end': 'mileMarkerEnd4',
-        'directions_of_traffic': 'directionOfTraffic4',
-        'current_posted_speed': 'currentPostedSpeed4',
-        'reduced_speed_limit': 'requestedTemporarySpeed4',
-        'start_date': 'workStartDate4',
-        'end_date': 'workEndDate4',
-    }
+        "street_name": "streetNameFrom4",
+        "mile_marker_start": "mileMarkerStart4",
+        "mile_marker_end": "mileMarkerEnd4",
+        "directions_of_traffic": "directionOfTraffic4",
+        "current_posted_speed": "currentPostedSpeed4",
+        "reduced_speed_limit": "requestedTemporarySpeed4",
+        "start_date": "workStartDate4",
+        "end_date": "workEndDate4",
+    },
 ]
 
 
 def main():
     navjoy_file, output_dir = parse_rtdh_arguments()
-    input_file_contents = open(navjoy_file, 'r').read()
-    generated_messages = generate_standard_messages_from_string(
-        input_file_contents)
+    input_file_contents = open(navjoy_file, "r").read()
+    generated_messages = generate_standard_messages_from_string(input_file_contents)
 
     generated_files_list = []
     for message in generated_messages:
         output_path = f"{output_dir}/standard_568_{message['event']['source']['id']}_{round(message['rtdh_timestamp'])}_{message['event']['detail']['direction']}.json"
-        open(output_path, 'w+').write(json.dumps(message, indent=2))
+        open(output_path, "w+").write(json.dumps(message, indent=2))
         generated_files_list.append(output_path)
 
     if generated_files_list:
-        print(
-            f"Successfully generated standard message files: {generated_files_list}")
+        print(f"Successfully generated standard message files: {generated_files_list}")
```

Comment:
Print to logging swap

Reply:
Updated other logging statements to use print